src/sidebar/store/test/index-test.js

```diff
@@ -19,172 +20,206 @@ const fixtures = immutable({
 });
 
 describe('store', function() {
-  let store;
+  let fakeDefaultKeys;
+  let fakeLocalStorage;
   let fakeRootScope;
 
-  function tagForID(id) {
-    const storeAnn = store.findAnnotationByID(id);
-    if (!storeAnn) {
-      throw new Error(`No annotation with ID ${id}`);
-    }
-    return storeAnn.$tag;
-  }
-
   beforeEach(function() {
+    fakeDefaultKeys = {};
+    fakeLocalStorage = { getItem: sinon.stub() };
     fakeRootScope = { $applyAsync: sinon.stub() };
-    store = storeFactory(fakeRootScope, {});
+
+    $imports.$mock({
+      '../util/default-keys': fakeDefaultKeys,
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
   });
 
   describe('initialization', function() {
     it('does not set a selection when settings.annotations is null', function() {
+      const store = storeFactory(fakeRootScope, fakeLocalStorage, {});
       assert.isFalse(store.hasSelectedAnnotations());
       assert.equal(Object.keys(store.getState().selection.expanded).length, 0);
     });
 
     it('sets the selection when settings.annotations is set', function() {
-      store = storeFactory(fakeRootScope, { annotations: 'testid' });
+      const store = storeFactory(fakeRootScope, fakeLocalStorage, {
+        annotations: 'testid',
+      });
       assert.deepEqual(store.getState().selection.selectedAnnotationMap, {
         testid: true,
       });
     });
 
     it('expands the selected annotations when settings.annotations is set', function() {
-      store = storeFactory(fakeRootScope, { annotations: 'testid' });
+      const store = storeFactory(fakeRootScope, fakeLocalStorage, {
+        annotations: 'testid',
+      });
       assert.deepEqual(store.getState().selection.expanded, {
         testid: true,
       });
     });
-  });
 
-  describe('clearSelection', () => {
-    // Test clearSelection here over the entire store as it triggers the
-    // CLEAR_SELECTION action in multiple store modules.
-    it('sets `selectedAnnotationMap` to null', () => {
-      store.clearSelection();
-      assert.isNull(store.getState().selection.selectedAnnotationMap);
-    });
+    it('retrieves known defaults from persisted local storage', () => {
+      //fakeDefaultKeys = { foo: 'bar' };
+      storeFactory(fakeRootScope, fakeLocalStorage, {});
 
-    it('sets `filterQuery` to null', () => {
-      store.clearSelection();
-      assert.isNull(store.getState().selection.filterQuery);
+      assert.calledOnce(fakeLocalStorage.getItem);
+      //assert.calledWith(fakeLocalStorage.getItem, 'bar');
     });
+  });
 
-    it('sets `directLinkedGroupFetchFailed` to false', () => {
-      store.clearSelection();
-      assert.isFalse(
-        store.getState().directLinked.directLinkedGroupFetchFailed
-      );
-    });
+  describe('FIXME: move into appropriate store module tests', () => {
```

Comment:
I've corralled these here because these tests assume a pre-initialized `store` from their `beforeEach`, but I don't want that to be the case for the actual meaningful initialization tests in this test module. I didn't want to take the time _just now_ to move these tests to their appropriate sub-module, but it should happen soon.

Diffs in this file below here are not meaningful.